docker/aligned_base.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:bookworm-slim
+
+RUN apt update -y && apt upgrade -y
+
+# Install golang 1.22.2
+ENV GO_VERSION=1.22.2
+RUN apt install -y wget \
+                   tar \
+                   curl \
+                   git \
+                   make \
+                   clang \
+                   pkg-config \
+                   openssl \
+                   libssl-dev \
+                   yq \
+                   jq
+RUN wget https://golang.org/dl/go$GO_VERSION.linux-amd64.tar.gz
+RUN tar -C /usr/local -xzf go$GO_VERSION.linux-amd64.tar.gz
```

Comment:
I believe this should use whatever the host arch is.

Reply:
Fixed!